### PR TITLE
[Merged by Bors] - feat(Probability/Kernel/Basic): Evaluation function of a finite kernel is integrable

### PR DESCRIPTION
**Pull request by kex-y**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by RemyDegenne** [comment]
Can you merge master, to see if it fixes the stuck "detect changes to headers SHAs" test?

File: Mathlib/Probability/Kernel/Basic.lean [hunk 1]
```diff
@@ -213,6 +213,21 @@ protected theorem measurable_coe (κ : kernel α β) {s : Set β} (hs : Measurab
   (Measure.measurable_coe hs).comp (kernel.measurable κ)
 #align probability_theory.kernel.measurable_coe ProbabilityTheory.kernel.measurable_coe
 
+lemma IsFiniteKernel.integrable (μ : Measure α) [IsFiniteMeasure μ]
+  (κ : kernel α β) [IsFiniteKernel κ] {s : Set β} (hs : MeasurableSet s) :
```
Comment (RemyDegenne) [inline Mathlib/Probability/Kernel/Basic.lean:217]:
```suggestion
    (κ : kernel α β) [IsFiniteKernel κ] {s : Set β} (hs : MeasurableSet s) :
```

File: Mathlib/Probability/Kernel/Basic.lean [hunk 2]
```diff
@@ -213,6 +213,21 @@ protected theorem measurable_coe (κ : kernel α β) {s : Set β} (hs : Measurab
...
+    Integrable (fun x => (κ x s).toReal) μ := by
+  refine' Integrable.mono' (integrable_const (IsFiniteKernel.bound κ).toReal)
+    ((kernel.measurable_coe κ hs).ennreal_toReal.aestronglyMeasurable)
+    (ae_of_all μ <| fun x => _)
+  rw [Real.norm_eq_abs, abs_of_nonneg ENNReal.toReal_nonneg,
+    ENNReal.toReal_le_toReal (measure_ne_top _ _) (IsFiniteKernel.bound_ne_top _)]
+  exact kernel.measure_le_bound _ _ _
+
+lemma IsMarkovKernel.integrable (μ : Measure α) [IsFiniteMeasure μ]
+  (κ : kernel α β) [IsMarkovKernel κ] {s : Set β} (hs : MeasurableSet s) :
```
Comment (RemyDegenne) [inline Mathlib/Probability/Kernel/Basic.lean:227]:
```suggestion
    (κ : kernel α β) [IsMarkovKernel κ] {s : Set β} (hs : MeasurableSet s) :
```

**Review by RemyDegenne — APPROVED** [review]
LGTM
bors d+

**Comment by RemyDegenne** [comment]
Did you merge the wrong branch? There are now category theory changes in this PR.

**Comment by JasonKYi** [comment]
Yeah, I merged the wrong branch. I'm trying to figure out how to fix this

**Pull request closed without merging**